client/src/pages/cejst.tsx

```diff
@@ -50,7 +48,7 @@ const CEJSTPage = ({location}: IMapPageProps) => {
           </p>
         </Alert>
         <h2>Explore the Tool</h2>
-        <MapWrapper features={features} />
+        <MapWrapper/>
```

Comment:
So, you're not passing in any `features` prop, but the MapWrapper checks the length of this prop. Does typescript/react just assume a default, empty `Feature<Geometry>[]`?
(I don't thing this is not a bug, I'm just doing a visual code inspection and my in-brain-compiler doesn't know what's happening here.)
Feel free to just chat with me in person instead of answering this in the PR.

Reply:
Yeah, this originally came from [here](https://taylor.callsen.me/using-openlayers-with-react-functional-components/) - the idea is that you can assign a `features` prop with an array of features, in this case from a geoJSON file, which will be added to the map - this could be useful say if we had a dropdown with the ability to select a layer, the output of which would be a geoJSON representation; we don't yet have this use case, but if/when we do this will support it. Notably it may make sense to remove this until that is the case (and that may never be able to be the case given our features are not represented at the moment with geoJSON)